models/place-videos.json

```diff
@@ -0,0 +1,19 @@
+{
+  "title": "place.videos",
+  "type": "array",
+  "description": "Describes an array of video objects that are linked to a place.",
+  "examples": [
+    {
+      "id": "b504cf44-9ab8-4641-9934-38d1cc67242c",
+      "url": "https://www.youtube.com/?v=ewr53d",
+      "copyrightHolder": "publiq"
+    },
+    {
+      "id": "46e9ea9f-fc42-4759-a81b-4308467b7c35",
+      "url": "https://www.vimeo.com/54fdrg"
+    }
+  ],
```

Comment:
These examples are technically incorrect (also for `event-videos.json`). Every entry in `examples` is a single separate examples of how the JSON can look like. Since this should be an array, it should either be:

```
  "examples": [
    [
        {
          "id": "b504cf44-9ab8-4641-9934-38d1cc67242c",
          "url": "https://www.youtube.com/?v=ewr53d",
          "copyrightHolder": "publiq"
        },
        {
          "id": "46e9ea9f-fc42-4759-a81b-4308467b7c35",
          "url": "https://www.vimeo.com/54fdrg"
        }
    ]
]
```
(one example with two videos)

or:

```
  "examples": [
    [
        {
          "id": "b504cf44-9ab8-4641-9934-38d1cc67242c",
          "url": "https://www.youtube.com/?v=ewr53d",
          "copyrightHolder": "publiq"
        }
    ],
    [
        {
          "id": "46e9ea9f-fc42-4759-a81b-4308467b7c35",
          "url": "https://www.vimeo.com/54fdrg"
        }
    ]
]
```
(two examples with one video each)

Reply:
You can also see this if you look at https://docs.publiq.be/docs/uitdatabank/branches/feature%2FIII-4232-add-video-docs/models/event-videos.json and click on the Example tabs. The property is documented as being an array but the examples are single objects.

Reply:
Clear! I added square brackets and added an extra example with multiple videos.